lib/ui/views/home/home_viewmodel.dart

```diff
@@ -1,13 +1,22 @@
 import 'package:filledstacked_academy/app/app.locator.dart';
 import 'package:filledstacked_academy/app/app.logger.dart';
 import 'package:filledstacked_academy/app/app.router.dart';
+import 'package:filledstacked_academy/enums/bottom_sheet_type.dart';
+import 'package:filledstacked_academy/enums/sign_in_result.dart';
+import 'package:filledstacked_academy/models/user/user.dart';
+import 'package:filledstacked_academy/services/user_service.dart';
 import 'package:filledstacked_academy/ui/views/home/home_view.form.dart';
 import 'package:stacked/stacked.dart';
 import 'package:stacked_services/stacked_services.dart';
 
 class HomeViewModel extends FormViewModel {
-  final log = getLogger('HomeViewModel');
+  final _log = getLogger('HomeViewModel');
```

Comment:
I like to keep this `log` instead of `_log` because when you write in other languages it's always `console.i` or `print` I don't think having it private makes it better. the typing of it will be more cumbersome.

Reply:
I just like it to be private because when you are using the viewmodel, in the autocompletion you can see the log exposed and I don't like that too much. But... nvm, is more or less the same.